auth/src/AzureSubscription.ts

```diff
@@ -55,4 +56,9 @@ export interface AzureSubscription {
      * The credential for authentication to this subscription. Compatible with Azure track 2 SDKs.
      */
     readonly credential: TokenCredential;
+
+    /**
+     * The account associated with this subscription. This is optional as we only need the account if there are duplicate subscriptions.
+     */
+    readonly account?: vscode.AuthenticationSessionAccountInformation;

```

Comment:
Why is this optional? It seems like it's always set.

Reply:
I think I originally had this optional since we only wanted to pass in the account when there was a duplicate but since then we changed the logic to always pass the account. I'll make this a required property.

Reply:
Actually looking further into it `AzureDevOpsSubscriptionProvider` uses a version of `AzureSubscription` without the account attached to it. So if i make it not optional it will break this and I'm not sure if that is something we would want to change.

Reply:
Ok, I'll look into it.